### PR TITLE
Removing loss weights

### PR DESCRIPTION
**Pull request by TonyCongqianWang**
I accidentally removed those params in #329. To make sure that the gain wasn't caused by this, it should be tasted in isolation..

### CODE REVIEW

**Comment by vondele** [comment]
cscs-ci run RECIPE=threats

**Comment by vondele** [comment]
cscs-ci run RECIPE=threats

**Comment by TonyCongqianWang** [comment]
Meh crashed again.. and seems like it is not resuming from checkpoint... Is there something we could do about that? I think there is a checkpoint every 10 epochs, right?

**Comment by vondele** [comment]
No, we have --max_epochs=800 --network-save-period=800 ...
When a job gets killed in this way it doesn't save the network (while time limit seem to do it). 
We can't save every 20 steps for both performance and space. Probably, with a reasonable frequency and only saving the last 1-2 checkpoints, not keeping all saved.

**Comment by vondele** [comment]
cscs-ci run RECIPE=threats

**Comment by vondele** [comment]
> Meh crashed again.. and seems like it is not resuming from checkpoint... Is there something we could do about that? I think there is a checkpoint every 10 epochs, right?

I've now implemented saving of additional checkpoints, which should help with this. Needs updated nnue-pytorch and nettest.

**Comment by vondele** [comment]
@TonyCongqianWang the trainer you use should also be update to include the corresponding trainer changes I merged.

**Comment by TonyCongqianWang** [comment]
Ahh.... well that kinda sucks as it would restart all training :/

So should I revert the merge?

**Comment by TonyCongqianWang** [comment]
Reverted the merge...

**Comment by vondele** [comment]
cscs-ci run RECIPE=threats

**Comment by TonyCongqianWang** [comment]
Ok this seems to be worse than the normal recipe.

Though that means the efficient_swa recipe could maybe be improved further by readding the scaling! There is somehow a mechanism that makes both anti-synergize. Hard to tell. Looking forward to the hyperparameter optimization
